src/ripple/app/misc/impl/Manifest.cpp

```diff
@@ -193,6 +192,18 @@ ManifestCache::applicable (
         return ManifestDisposition::untrusted;
     }
 
+    return applyTrustedManifest (std::move(m));
```

Comment:
Whats the motivation for adding the moves/value semantics rather than the `Manifest const &` you used before?